torch/nn/functional.py

```diff
@@ -708,15 +712,19 @@ def threshold(input, threshold, value, inplace=False):
 """)
 
 
+@torch._jit_internal.weak_script
 def relu(input, inplace=False):
+    # type: (Tensor, bool) -> Tensor
     r"""relu(input, inplace=False) -> Tensor
 
     Applies the rectified linear unit function element-wise. See
     :class:`~torch.nn.ReLU` for more details.
     """
     if inplace:
-        return torch.relu_(input)
-    return torch.relu(input)
+        result = torch.relu_(input)
```

Comment:
We should record handling more complicated return statement configurations as a feature task at p2.